packages/compass-explain-plan/src/modules/explain.spec.js

```diff
@@ -115,7 +117,12 @@ describe('explain module', () => {
   describe('#reducer', () => {
     context('when the action is not presented in the explain module', () => {
       it('returns the default state', () => {
-        expect(reducer(undefined, { type: 'test' })).to.deep.equal({
+        const result = reducer(undefined, { type: 'test' });
+
+        // resultId is a random number
+        expect(result.resultId).to.be.a('number');
+
+        expect(omit(result, 'resultId')).to.deep.equal({
```

Comment:
You can use `include` here and in other similar places

```suggestion
        expect(result).to.include({
```

Reply:
Yes, but then it is a partial match and will allow all unknown properties which is more than I want.